### PR TITLE
lab 7, lab 8

### PR DESCRIPTION
**Pull request by jackamoss2**
added functionality for duplicate name contacts

### CODE REVIEW

**Comment by jackamoss2** [comment]
to change:
- move save point to immediately after modify actions
- prevent program crash if nothing entered for name requests

**Review by brucestull — CHANGES_REQUESTED** [review]
[Lab 8: Contact List](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/08%20Contact%20List.md)

* Needs Work
* Missing file:
* Sample output:
```
PS C:\Users\Bruce\Programming\class_swordfish\code\jack\python> python .\lab08.py
Traceback (most recent call last):
  File "C:\Users\Bruce\Programming\class_swordfish\code\jack\python\lab08.py", line 5, in <module>
    with open('contacts.csv', 'r') as file:
FileNotFoundError: [Errno 2] No such file or directory: 'contacts.csv'
PS C:\Users\Bruce\Programming\class_swordfish\code\jack\python>
```

File: code/jack/python/lab08.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
+# Lab 08: Contact List
```
Comment (brucestull) [inline code/jack/python/lab08.py:1]: [Lab 8: Contact List](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/08%20Contact%20List.md)

Completed

File: code/jack/python/lab07.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import math
```
Comment (brucestull) [inline code/jack/python/lab07.py:1]: [Lab 7: Compute Automated Readability Index](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/07%20ARI.md)

* Needs Work
    * Missing sample file.

* Sample output:
```
PS C:\Users\Bruce\Programming\class_swordfish\code\jack\python> python .\lab07.py
Traceback (most recent call last):
  File "C:\Users\Bruce\Programming\class_swordfish\code\jack\python\lab07.py", line 6, in <module>
    with open(file_name, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'gettysburg_address.txt'
PS C:\Users\Bruce\Programming\class_swordfish\code\jack\python>
```

**Review by brucestull — CHANGES_REQUESTED** [review]
[Lab 7: Compute Automated Readability Index](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/07%20ARI.md) - Needs Work
[Lab 8: Contact List](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/08%20Contact%20List.md) - Completed

File: code/jack/python/lab07.py (hunk 2 above)
Comment (jackamoss2) [reply code/jack/python/lab07.py:1]: Sample file for lab 7 added

**Review by brucestull — APPROVED** [review]
[Lab 7: Compute Automated Readability Index](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/07%20ARI.md)

Completed

**Pull request merged**